lib/index.js

```diff
@@ -44,7 +45,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.post(
     basePath,
     optionallyAuthenticated,
-    validate('account.postAccounts'),
+    validate('account.create'),
```

Comment:
I missed this earlier.  We have been using a pattern with validator naming.  Not sure how good it is but it's at least consistent:  `domain.verbResource[Query]`.  Theory long ago (or at least mine) was that this would be a clear direct endpoint to schema name mapping using the verb (GET, POST, etc), the resource (Account, Accounts, etc), and either if not the body then add `Query`.  This avoids any conflicts as more endpoints are added.  Is there a reason to use another pattern?